plugins/main/public/components/common/wazuh-discover/render-columns.tsx

```diff
@@ -13,7 +13,7 @@ export const MAX_ENTRIES_PER_QUERY = 10000;
 
 const renderRequirementsSecurityOperations = (value: []) => {
   return (
-    <EuiText gutterSize='s' direction='column'>
+    <EuiText style={{ fontSize: '13px' }} gutterSize='s' direction='column'>
```

Comment:
**nitpick**: replace the usage of `EuiText` by returning the raw text joined by comma. We should ensure the value is an array before using the `.join` method.